actions/retest/go.mod

```diff
@@ -1,16 +1,16 @@
 module github.com/ceph/ceph-csi/actions/retest
 
-go 1.17
+go 1.18
```

Comment:
The dependencies were not updated because the Go version was not changed? Not sure why this commit is part of a protobuf rebase... This is a CI change, and probably should have been a separate PR in any case.

Reply:
@nixpanic seperated to new PR https://github.com/ceph/ceph-csi/pull/3528